_data/simple-icons.json

```diff
@@ -16158,6 +16158,12 @@
             "source": "https://tripadvisor.mediaroom.com/logo-guidelines",
             "guidelines": "https://tripadvisor.mediaroom.com/logo-guidelines"
         },
+        {
+            "title": "trivago",
+            "hex": "E32851",
```

Comment:
Any reason for picking this color in particular? I'd say the blue is more prominent if anything, though I'm curious to hear your thoughts! The brand guidelines don't say much about *which* of three primary colors to use as far as I can tell ahah.

Reply:
I chose red because it reflects the color used for 'tri' in their wordmark logo, and according to [this page](https://cloud.brandmaster.com/point/en/trivagobrandhub/component/default/119357)], red is a good choice for primary colors in single pairings.

Reply:
> according to [this page](https://cloud.brandmaster.com/point/en/trivagobrandhub/component/default/119357), red is a good choice for primary colors in single pairings.

That's what I thought at first, though if you read it carefully that is an *example* and the text below each part says "Single color" (not red specifically), the next one says "Either 2 primary colors or one
primary paired with Warm White" (also not red specifically) etc.